### PR TITLE
added new resource in data science

### PR DESCRIPTION
**Pull request by kraticodes**
## What does this PR do?
Added resource

### Description
Data science course (free)
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by kraticodes** [comment]
plz help me with the conflicting file issue

**Comment by eshellman** [comment]
please resolve conflicts.

**Comment by davorpa** [comment]
> plz help me with the conflicting file issue

Some ways to solve PR merge conflicts:
- [Github docs: Resolving a merge conflict on GitHub](https://docs.github.com/en/github/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github)
- Make a upstream master merge
- Use [Github web based editor](https://docs.github.com/es/codespaces/the-githubdev-web-based-editor#opening-the-web-based-editor) to solve it.

Next time, I recomend you create branches instead work on main/master.

**Comment by kraticodes** [comment]
plz have a look!

File: more/free-programming-interactive-tutorials-en.md [hunk 1]
```diff
@@ -111,6 +111,7 @@
 
 ### Data Science
 
+* [Data Science Foundations - IBM SkillBuild](https://skillsbuild.org/students/course-catalog/data-science) *(Email login reqired to access the course)*
```
Comment (davorpa) [inline more/free-programming-interactive-tutorials-en.md:114]: An insignificant misspelling in notes: `reqired`

but better normalize format according to CONTRIBUTING guidelines.

`*(email address required)*`

:rocket::rocket:

**Review by davorpa — CHANGES_REQUESTED** [review]
**TIP**: Rerequest a review when done, sometimes no needs make comments. Use icon buttons placed near who review to do this :wink:

**Pull request merged**